backend/oasst_backend/utils/hugging_face.py

```diff
@@ -8,10 +8,14 @@
 
 
 class HfUrl(str, Enum):
-    HUGGINGFACE_TOXIC_ROBERTA = ("https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta",)
+    HUGGINGFACE_TOXIC_ROBERTA = "https://api-inference.huggingface.co/models/unitary"
```

Comment:
this string value has nothing to do with "toxic" or "roberta", so it shouldn't be named that